migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -49,19 +50,30 @@ The following `imagestreamtags` have been _removed_ from {product-title} 4.2:
 
 include::modules/migration-understanding-cam.adoc[leveloffset=+1]
 
-== Installing the CAM Operator
+== Deploying the Cluster Application Migration tool
 
-You must install the CAM Operator xref:installing-migration-operator-manually_{context}[manually on the {product-title} 3.x source cluster] and xref:installing-migration-operator-with-olm_{context}[with OLM on the {product-title} 4.2 target cluster].
+Deploying the CAM tool involves the following steps:
 
-include::modules/migration-installing-migration-operator-manually.adoc[leveloffset=+2]
-include::modules/migration-installing-migration-operator-olm.adoc[leveloffset=+2]
+. Installing the CAM Operator on all clusters
```

Comment:
Not critical, but just wondering if these should link to the appropriate below sections.